src/EpisodeIntegrationService/ReceiveData/ReceiveData.cs

```diff
@@ -23,34 +23,12 @@ public async Task Run(
     {
         _logger.LogInformation($"C# Blob trigger function Processed blob\n Name:{name} \n Size: {myBlob.Length} Bytes");
 
-        // Validate JSON
         using (var reader = new StreamReader(myBlob))
         {
-            var jsonData = reader.ReadToEnd();
-            if (IsValidJson(jsonData))
-            {
-                _logger.LogInformation("JSON is valid.");
-                await _httpRequestService.SendPost(Environment.GetEnvironmentVariable("ProcessDataURL"), jsonData);
-            }
-            else
-            {
-                _logger.LogError("JSON is invalid.");
-            }
-        }
-    }
+            var csvData = reader.ReadToEnd();
 
-    private bool IsValidJson(string jsonData)
-    {
-        try
-        {
-            _logger.LogInformation("JSON is valid:{jsonData}", jsonData);
-            var obj = JsonSerializer.Deserialize<object>(jsonData);
-            return obj != null;
-        }
-        catch (JsonException)
-        {
-            _logger.LogError("Could not validate JSON");
-            return false;
+            _logger.LogInformation("Sending CSV data to the ProcessData function");
+            await _httpRequestService.SendPost(Environment.GetEnvironmentVariable("ProcessDataURL") + $"?FileName={name}", csvData);
```

Comment:
Need some exception handling in this method

Reply:
added this.